src/Letter.ts

```diff
@@ -0,0 +1,119 @@
+import type { TDocumentDefinitions, ContentText, ContentImage, Content } from 'pdfmake/interfaces';
+import { layout as dinLayoutFunction } from './layouts/din-5008-a';
+import { formatAddress, mm2pt, stripTags, applyFunctionToPdfMakeContent } from './utility';
+import type { LayoutFunction, PdfMakeContent, Signature } from './types';
+
+type LetterProps = {
+    sender_address: string[] | string;
+    recipient_address: string[] | string;
+    /** The information block usually displayed at the top-right of the letter. */
+    information_block?: PdfMakeContent;
+    subject: string;
+    content: PdfMakeContent;
+    signature: Signature;
+};
+// The `marginTop` is missing in the type definitions.
+type PdfMakeSignature = (ContentText | ContentImage) & { marginTop: number };
```

Comment:
Did you open a pull request? Might be worth fixing at the source.

Reply:
Setting `marginTop` does work but it seems like more of an internal (or maybe deprecated?) property. It's not mentioned anywhere in the docs.

Reply:
Well, that sounds unsettling… but ok